ena_upload/ena_upload.py

```diff
@@ -101,23 +102,26 @@ def check_columns(df, schema, action, dev, auto_action):
                     for index, row in df.iterrows():
                         remote_present = np.nan
                         try:
-                            remote_present = str(identify_action(
-                                schema, str(df['alias'][index]), dev)).upper()
+                            remote_present = remote_check(
+                                schema, str(df['alias'][index]), dev)
 
                         except Exception as e:
                             print(e)
                             print(
                                 f"Something went wrong with detecting the ENA object {df['alias'][index]} on the servers of ENA. This object will be skipped.")
-                        if remote_present == np.nan:
```

Comment:
that nan check is not needed anymore?

Reply:
the function that fills in remote_present can only contain a boolean, so not np.nan

Reply:
and is now caught by the `else` below